lib/msal-common/src/authority/Authority.ts

```diff
@@ -438,33 +443,48 @@ export class Authority {
      * @param newMetadata
      */
     private async updateCloudDiscoveryMetadata(metadataEntity: AuthorityMetadataEntity): Promise<AuthorityMetadataSource> {
+        this.logger.verbose("Attempting to get cloud discovery metadata in the config");
         let metadata = this.getCloudDiscoveryMetadataFromConfig();
         if (metadata) {
+            this.logger.verbose("Found cloud discovery metata in the config.");
             metadataEntity.updateCloudDiscoveryMetadata(metadata, false);
             return AuthorityMetadataSource.CONFIG;
         }
+        this.logger.verbose("Did not find cloud discovery metadata in the config");
```

Comment:
nit: On this line and similar ones below, I would remove this log since if we move to the next search method, we already know the previous one failed.
Also in general I'd combine the logger calls, which are one after the other, into one string and one call to improve perf.